src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,50 @@
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export class Tab extends Component {
+  state = {
+    currentTab: this.props.tabs[0].title,
+  }
+
+  tabToggler = (tab) => {
+    this.setState({ currentTab: tab.title });
+  }
+
+  render() {
+    const { tabs } = this.props;
+
+    return (
+      <div className="tabs">
+        <div className="tabs__tabrow">
+          {tabs.map(tab => (
+            <button
+              key={tab.title}
+              className={classNames({
+                tabs__btn: true,
+                tabs__btn_active: tab.title === this.state.currentTab,
+              })}
+              type="button"
+              onClick={() => {
+                this.tabToggler(tab);
```

Comment:
```suggestion
                this.tabToggler(tab.title);
```
You can pass here only 'title', not the whole 'tab' object (without 'content' field)
And your 'tabToggler' function should look like this:
```
 tabToggler = (title) => {
    this.setState({ currentTab: title });
  }
```